package.json

```diff
@@ -50,6 +50,7 @@
     "nunjucks": "^3.0.1",
     "postcodesio-client": "billinghamj/postcodesio-client-node",
     "require-environment-variables": "^1.1.2",
+    "tunnel-agent": "^0.6.0",
```

Comment:
It's a dependency of node-sass that snyk was complaining about. I guess it should be fixed another way. Will take a look.